lms/envs/aws.py

```diff
@@ -676,6 +676,12 @@
         'courseware.student_field_overrides.IndividualStudentOverrideProvider',
     )
 
+##### Self-Paced Course Due Dates #####
+if FEATURES.get('ENABLE_SELF_PACED_COURSES'):
+    FIELD_OVERRIDE_PROVIDERS += (
+        'courseware.self_paced_overrides.SelfPacedDateOverrideProvider',
+    )
```

Comment:
Since the override provider is only `enabled` for courses marked as currently self-paced, couldn't we always configure the provider?